### ISSUE TITLE
Cannot scroll on /archives

### ISSUE THREAD

**Issue by MadLittleMods** [gitterHQ/gitter#1320]
As reported by @andersevenrud

There is no scrolling for the overflow on `/archives`, https://gitter.im/gitterHQ/gitter/archives

We link to `/archives/all` from the dropdown menu, https://gitter.im/gitterHQ/gitter/archives/all
